modules/rubiconBidAdapter.js

```diff
@@ -489,6 +489,15 @@ export const spec = {
       });
     }
 
+    /**
+     * Prebid AdSlot
+     * @type {(string|undefined)}
+     */
+    const pbAdSlot = utils.deepAccess(bidRequest, 'context.pbAdSlot');
+    if (typeof pbAdSlot === 'string' && pbAdSlot) {
+      data['tg_i.dfp_ad_unit_code'] = pbAdSlot.replace(/^\/+/, '');
+    }
+
```

Comment:
doesn't this new param need to be added to the "priority" list of params?

Reply:
doesn't it also have to be added to the video request?

Reply:
All `tg_i.*` params are automatically added, on line 331 `const containsTgI = /^tg_i/` .
I'll add to the video request